### PR TITLE
Style HotTopics buttons

### LINKED ISSUE #35: Create CSS/Sass in Topics Component

### PR DESCRIPTION
**Pull request by camianderson**
Hi Team, Please review this PR. 

## Description of Changes:
* Add styling to HotTopic buttons
* Style Topics

## Put an 'X' next to all that apply
New Test: []
New Feature:[x]
Bug Fix: []

## Before You Submit Ensure
All Tests Pass: [x]
Full Coverage: [x]
Runs Locally: [x]

Closes #35 #30 

Thanks! 

-Cami

### CODE REVIEW

File: src/components/Topics/Topics.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(235, 231, 146, 0.8);
+  border-radius: 3px;
+  font-size: 1.2rem;
+  box-shadow: 0 1.5rem 4rem rgba(20, 32, 40, 0.2);
+}
+.topics .topics__box:hover {
```
Comment (jordan-sullivan) [inline src/components/Topics/Topics.css:18]: nice hover effect

File: src/components/Topics/Topics.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  height: 10rem;
+  padding: 0.5rem;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(235, 231, 146, 0.8);
+  border-radius: 3px;
+  font-size: 1.2rem;
+  box-shadow: 0 1.5rem 4rem rgba(20, 32, 40, 0.2);
```
Comment (jordan-sullivan) [inline src/components/Topics/Topics.css:16]: nice, love how subtle this is

**Review by jordan-sullivan — APPROVED** [review]
Great job getting it back to a good spot Cami -J

**Review by Neil-B-Linton — APPROVED** [review]
Great work with the styling, love how responsive the buttons are

**Pull request merged**